requirements.txt

```diff
@@ -1,12 +1,23 @@
+# Python version requirement: >=3.11 (see pyproject.toml)
+# Note: TCREMP requires Python 3.11+, other models work with older versions
+
 numpy
 pandas
 torch
 transformers
 typer
 antiberty
+ablang
 rjieba
 pre-commit
 protobuf
 sentencepiece
 pytest-workflow>=1.6.0
 pytest>=7.0.0
+gensim>=3.8.3
+
+
+# TCR embedding tools (require manual installation)
+# TCREMP: Requires Python 3.11+ (specified above)
+#   git clone https://github.com/antigenomics/tcremp.git && cd tcremp && pip install .
+# TCR-BERT: Available through transformers (wukevin/tcr-bert)
```

Comment:
we should move the information on tools that require manual installation to the documentation section that explains all the tools.